docs/spec/fastsync/fastsync.md

```diff
@@ -0,0 +1,1101 @@
+# Fastsync
+
+<!--
```

Comment:
Maybe we should remove these comments from the template.

Reply:
I removed them in a previous revision